src/server/project.js

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var fs = require('fs'),
+    path = require('path'),
+    cordovaServe = require('cordova-serve'),
+    Q = require('q'),
+    dirs = require('./dirs'),
+    pluginUtil = require('./utils/plugins'),
+    prepareUtil = require('./utils/prepare'),
+    utils = require('./utils/jsUtils');
+
+/**
+ * @constructor
+ */
+function Project(simulator, platform) {
+    this._simulator = simulator;
```

Comment:
The `_simulator` is used in this file only for its `telemetry` and `config` properties. Instead of passing the `_simulator` object here, can we pass only what is needed - its `telemetry` and `config` properties?